lib/middleware.js

```diff
@@ -309,6 +311,12 @@ var middlewares = module.exports = {
   },
   respondImages: function (req, res) {
     var data = Object.keys(images).map(function (id) {
+      // if we have full image info, we can return that instead
+      // assume if container_config exist on image, it is full object
+      if (images[id].container_config) {
+        return images[id];
+      }
+      // fake return object if we do not have all info
```

Comment:
I didn't see this before. I don't think this will include `RepoTags` if you take this short cut. is that true?